build.gradle

```
@@ -14,6 +14,7 @@ plugins {
     id 'com.diffplug.spotless' version '6.0.0'
     id 'com.github.ben-manes.versions' version '0.39.0'
     id 'ru.vyarus.mkdocs' version '3.0.0'
+    id 'edu.sc.seis.launch4j' version '3.0.3'
```

Bump edu.sc.seis.launch4j@3.0.5

```suggestion
    id 'edu.sc.seis.launch4j' version '3.0.5'
```